<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog4</class>
 <widget class="QDialog" name="Dialog4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BuyButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>281</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="Size1_2">
        <property name="text">
         <string>35</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size2_2">
        <property name="text">
         <string>36</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size3_2">
        <property name="text">
         <string>37</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size4_2">
        <property name="text">
         <string>38</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size5_2">
        <property name="text">
         <string>39</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="Size6_2">
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size7_2">
        <property name="text">
         <string>41</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size8_2">
        <property name="text">
         <string>42</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size9_2">
        <property name="text">
         <string>43</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
